op-proposer/proposer/l2_output_submitter.go

```diff
@@ -13,6 +13,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/ethereum-optimism/optimism/op-proposer/metrics"
```

Comment:
import should be with the rest of the local imports

Reply:
just fixed this! thanks for catching this.